storage_service/common/utils.py

```diff
@@ -351,63 +349,10 @@ def coerce_str(string):
     return string
 
 
-def add_agents_to_event_as_list(event, agents):
-    """Add agents in ``agents`` to the list ``event`` which represents a
-    PREMIS:EVENT.
-    :param list event: a PREMIS:EVENT represented as a list
-    :param iterable agents: an iterable of premisrw.PREMISAgent instances.
-    """
-    for agent in agents:
-        event.append(
-            (
-                "linking_agent_identifier",
-                ("linking_agent_identifier_type", agent.identifier_type),
-                ("linking_agent_identifier_value", agent.identifier_value),
-            )
-        )
-    return event
-
-
 def mets_file_now():
     return datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
 
 
-def get_ss_premis_agents(inst=True):
```

Comment:
I missed this, but there are a couple of left-over references to this function in the code, e.g. when creating replicated packages: 
```python
/home/ross-spencer/git/artefactual-labs/am/src/archivematica-storage-service/storage_service/common/management/commands/import_aip.py:
  380      shutil.rmtree(compressed_aip_parent_path)
  381      aip_model_inst.size = utils.recalculate_size(new_full_path)
  382:     compression_agents = utils.get_ss_premis_agents()
  383      compression_event = premis.get_premis_aip_compression_event(
  384          details["event_detail"],

/home/ross-spencer/git/artefactual-labs/am/src/archivematica-storage-service/storage_service/locations/models/package.py:
 1142  
 1143          # 3. Construct the pointer file and return it
 1144:         replica_premis_creation_agents = utils.get_ss_premis_agents()
 1145          __, compression_program_version, archive_tool = (
 1146              master_compression_event.compression_details
 ....
 1198          old_premis_events = old_fsentry.get_premis_events()
 1199          old_premis_agents = old_fsentry.get_premis_agents()
 1200:         ss_agents = utils.get_ss_premis_agents()
 1201          replication_event = premis.get_replication_event(
 1202              self, replica_package, event_uuid=replication_event_uuid, agents=ss_agents
 ....
 1238          old_premis_agents = old_fsentry.get_premis_agents()
 1239          new_premis_events = list(set(old_premis_events + storage_effects.events))
 1240:         new_premis_agents = list(set(old_premis_agents + utils.get_ss_premis_agents()))
 1241          new_composition_level = old_composition_level
 1242          if storage_effects.composition_level_updater:
```

Reply:
😱 oops, that's no good. Thanks for catching! Updated now.